File: 1427.py
n = int(input())

A = []
while n > 0:
    A.append(n % 10)   # 각 자리수 리스트에 저장
    n = n // 10

A.sort()   # 리스트 정렬
ret = 0
d = 1
for i in A:
    ret += d * i   
    d = d * 10  

print(ret)

File: 1427.cpp
#include <iostream>
#include <vector>
#include <algorithm>
using namespace std;

int main()
{
	int n;
	cin >> n;

	vector<int> d;
	while (1) {
		if (n == 0)
			break;

		int k = n % 10;
		d.push_back(k);
		n = n / 10;
	}

	sort(d.begin(), d.end(), greater<int>());

	int ret = 0;
	int p = 1;
	while (d.size() != 0) {
		ret += d.back() * p;
		d.pop_back();
		p = p * 10;
	}

	cout << ret;

	return 0;
}
